lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -96,16 +104,11 @@ describe('ReportUIFeatures', () => {
 
   describe('initFeatures', () => {
     it('should init a report', () => {
-      // render a report onto the UIFeature dom
-      const container = reportUIFeatures._dom._document.querySelector('main');
-      renderer.renderReport(sampleResults, container);
-
-      assert.equal(reportUIFeatures.json, undefined);
-      reportUIFeatures.initFeatures(sampleResults);
-      assert.ok(reportUIFeatures.json);
```

Comment:
testing `json` was being used as a proxy test for the reportUIFeatures being initialized, otherwise this is just testing the report renderer.

We could definitely do something much better if you want to come up with something to test :) or just restore and we'll think about it another day

Reply:
How about just removing it? Doesn't seem useful now that we have all these other tests checking individual behaviors.

Reply:
> How about just removing it?

seems fine now, yeah